COPYING

```diff
@@ -1,14 +1,13 @@
-                  GNU LESSER GENERAL PUBLIC LICENSE
-                       Version 2.1, February 1999
+                  GNU LIBRARY GENERAL PUBLIC LICENSE
```

Comment:
@EmmanuelP: you now changed some of this from "lesser" (LGPL) to "library" (*not* LGPL) and others from "library" to "lesser". i don't think that this was intended since you previously stated that it should be the "lesser general public license 2.1 or later" (as also identified by your SPDX identifier)?

Reply:
Well, I did not pay attention to the library to lesser change, until I have modified almost 300 files. I don't think it matters that much, but I will probably switch everything to the 2.1 version, the lesser one.